### DISCUSSION TITLE
How to give parameters to the fast-downward planner

### DISCUSSION THREAD

**Discussion by Drejer1** [aiplan4eu/unified-planning#529] [Q&A]
When running the fast-downward I want it to have a time limit and what ever heuristics I want. I have looked at some of the examples that modify the keyword argument "params" but I struggle with matching it to the fast-downward planner.  

```python
with OneshotPlanner(name='fast-downward', params={'heuristic': 'lama-first'}) as planner:
```

**Comment by mikand** [comment]
@roeger maybe you can answer this?

**Comment by arbimo** [comment]
Actually, setting the time limit is independent of the planing engine and should be done with the `timeout` parameter of the `solve()` method. (but I have no clue regarding fast-downward's parameters)
